Scripts/Services/ExploringTheDeep/Items/WoodenKeyBarrel.cs

```diff
@@ -104,13 +113,19 @@ public override bool OnBeforeDestroyed()
         public override void Serialize(GenericWriter writer)
         {
             base.Serialize(writer);
-            writer.Write((int)0); //version
+            writer.Write((int)1); //version
+
+            writer.Write((int)this.m_key);
+            writer.Write(this.m_StorageLocker);
         }
 
         public override void Deserialize(GenericReader reader)
         {
             base.Deserialize(reader);
             int version = reader.ReadInt();
+
```

Comment:
Needs a version check here, if less than 1, do not read the new properties - give them default values instead.